### DISCUSSION TITLE
Is there any way to encrypt coder_parameters?

### DISCUSSION THREAD

**Discussion by maaaaaarkova** [coder/coder#8122] [General] [answered]
For example, I have a coder_parameter link to you git repo. And if I want to clone some private repo I want to use git clone https request with my credentials so i set my parameter value as https://username:password@domain/username/repo.git .
As any other parameter this one will be visible for anyone who has an access to my workspace. So I`d like to know if there is a way to hide my coder parameter in this case

**Comment by matifali** [comment, marked as answer]
It is not recommended to use `coder_parameter` for storing any secrets. 
For your use case, you should set up [`git_auth`](https://coder.com/docs/v2/latest/admin/auth#step-1-configure-the-oauth-application-in-github)
